src/ensembl/production/metadata/updater/core.py

```diff
@@ -342,7 +352,7 @@ def get_assembly_sequences(self, species_id, assembly):
                         name = preferred_name
 
                 # Combine all unique names with ";". If a name appears in multiple sequences with the same accession,
-        #        name = ";".join(info["names"])
+                #        name = ";".join(info["names"])
```

Comment:
If this comment is not meant to be retired soon, please simply delete the line, it's confusing for the future developer who might come after you.